docs/src/whatsnew/latest.rst

```diff
@@ -30,7 +30,9 @@ This document explains the changes made to Iris for this release
 ✨ Features
 ===========
 
-#. N/A
+#. `@jrackham-mo`_ added :meth:`~iris.io.format_picker.FormatAgent.copy` and
+   equality methods to :class:`iris.io.format_picker.FormatAgent`, as requested
+   in :issue:`6108`.
```

Comment:
Could you also add a link to this pull request here for ease of access?

```suggestion
   in :issue:`6108`, actioned in :pull:`6119`.
```

Reply:
Thanks @ESadek-MO that should now be fixed